zh/deploy-tidb-cluster-across-multiple-kubernetes.md

```diff
@@ -44,7 +46,11 @@ cluster1_name="cluster1"
 cluster1_domain="cluster1.com"
 # 集群 1 的命名空间
 cluster1_namespace="pingcap"
+```
 
+执行以下指令：
+
+```bash
```

Comment:
Global comment: add `{{< copyable "shell-regular" >}}` before each command that users want to copy and paste. For example:

```suggestion
{{< copyable "shell-regular" >}}

```bash
```

Reply:
fixed